application/config/autoload.php

```diff
@@ -132,5 +132,5 @@
 |
 |	$autoload['model'] = array('first_model' => 'first');
 */
-$autoload['model'] = array('flight', 'plane', 'flightsList', 'planesList');
+$autoload['model'] = array('flight', 'plane', 'flightsList', 'planesList', 'wackymodel');
```

Comment:
Think this should be `wackyModel` in order to access the `WackyModel` model.

Reply:
its working for me

Reply:
but can you pull and try just to make sure

Reply:
Just tested - app seems to be working fine. However, the travis build is complaining.
<img width="727" alt="wackymodel" src="https://user-images.githubusercontent.com/15661260/32697183-285f0ef6-c740-11e7-87fb-588bb6545d66.png">

Reply:
I see i guess its needed to be changed for the travis build